README.md

```diff
@@ -52,30 +52,71 @@ yet compatible with the golang BOSH v2.
 Required to create certificates. Requires golang.
 ```sh
 go get github.com/square/certstrap
+cd $GOPATH/src/github.com/square/certstrap
+go build
+go install
```

Comment:
`go get` should have done the `build` and `install` steps for you. `go get -d` would explicitly turn that off.